src/routes/controllers/utils/helper.js

```diff
@@ -0,0 +1,45 @@
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcryptjs';
+
+const secret = process.env.SECRET_KEY;
+
+const Help = {
```

Comment:
Let's not wrap our helper functions with `Help` Object, `helpers.js` is already serving as a namespace/module for these functions. 
We can export each of the individual functions as 
```javascript
export function hashPassword(password) {
   return bcrypt.hashSync(password, bcrypt.genSaltSync(10));
}
// OR
export const hashPassword = (password) => bcrypt.hashSync(password, bcrypt.genSaltSync(10));
```